app/src/main/java/com/strv/movies/MainActivity.kt

```diff
@@ -86,7 +165,30 @@ class MainActivity : ComponentActivity() {
 
 @Composable
 fun Greeting(name: String) {
-    Text(text = "Hello $name!")
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier.padding(start = 130.dp, bottom = 4.dp, top = 16.dp),
+    ) {
+        Row(
```

Comment:
Row not needed (only one element inside).

Reply:
done